samples/CommunityToolkit.Maui.Sample/ViewModels/Converters/IsNotNullConverterViewModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace CommunityToolkit.Maui.Sample.ViewModels.Converters;
+
+public class IsNotNullConverterViewModel : BaseViewModel
+{
+	int? intCheck;
+	public int? IntCheck
```

Comment:
let's use MVVM ObervableProperty attribute

Reply:
@VladislavAntonyuk thanks not seen that before